src/main/java/core/basesyntax/service/operations/SupplyOperationsHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
+
+public class SupplyOperationsHandler implements OperationsHandler {
+
+    @Override
+    public Operations getOperation() {
+        return Operations.SUPPLY;
+    }
+
+    @Override
+    public int getOperationsResult(int result, int quantity) {
+        return result + quantity;
```

Comment:
The method `getOperationsResult` should ensure that the result does not become negative after the operation. In the case of 'Supply', this might not be an issue, but it's good to be consistent across different operation handlers.